### PR TITLE
logging: clean up test

### PR DESCRIPTION
**Pull request by pongad**
no change in behavior

### CODE REVIEW

File: google-cloud-logging/src/test/java/com/google/cloud/logging/LoggingHandlerTest.java [hunk 1]
```diff
@@ -128,6 +127,8 @@
       .addLabel("levelValue", String.valueOf(LoggingLevel.EMERGENCY.intValue()))
       .setTimestamp(123456789L)
       .build();
+  private static final WriteOption[] DEFAULT_OPTION =
```
Comment (garrettjonesgoogle) [inline google-cloud-logging/src/test/java/com/google/cloud/logging/LoggingHandlerTest.java]: This is a list of options - call it `DEFAULT_OPTIONS`

Comment (pongad) [inline google-cloud-logging/src/test/java/com/google/cloud/logging/LoggingHandlerTest.java]: derp, done

**Comment by pongad** [comment]
@garrettjonesgoogle PTAL

**Comment by garrettjonesgoogle** [comment]
LGTM after >=1 tests pass

**Comment by pongad** [comment]
Everything passed except Mac which hasn't started. Merging.

**Pull request merged**